pkgs/applications/editors/vscode/extensions/python/default.nix

```
@@ -41,7 +48,34 @@ vscode-utils.buildVscodeMarketplaceExtension {
       --replace "\"default\": \"python\"" "\"default\": \"${python3.interpreter}\""
   '';
 
+  passthru.updateScript = writeScript "update" ''
+    #! ${bash}/bin/bash
+
+    set -eu -o pipefail
+
+    export PATH=${lib.makeBinPath [
+      curl
+      coreutils
+      gnused
+      nix
+    ]}
+
+    api=$(curl -s 'https://marketplace.visualstudio.com/_apis/public/gallery/extensionquery' \
+      -H 'accept: application/json;api-version=3.0-preview.1' \
+      -H 'content-type: application/json' \
+      --data-raw '{"filters":[{"criteria":[{"filterType":7,"value":"${mktplcRef.publisher}.${mktplcRef.name}"}]}],"flags":512}')
+    version=$(echo $api | sed -n -E 's|^.*"version":"([0-9.]+)".*$|\1|p')
+
+    if [[ $version != ${mktplcRef.version} ]]; then
+      tmp=$(mktemp)
+      curl -Lo $tmp $(echo ${(import ../mktplcExtRefToFetchArgs.nix mktplcRef).url} | sed "s|${mktplcRef.version}|$version|")
+      hash=$(nix hash file --type sha256 --base32 --sri $tmp)
+      sed -i -e "s|${mktplcRef.sha256}|$hash|" -e "s|${mktplcRef.version}|$version|" pkgs/applications/editors/vscode/extensions/python/default.nix
+    fi
+  '';
+
   meta = with lib; {
+    homepage = "https://marketplace.visualstudio.com/items?itemName=ms-python.python";
```

```suggestion
    downloadPage = "https://marketplace.visualstudio.com/items?itemName=ms-python.python";
    homepage = "https://github.com/Microsoft/vscode-python";
```

changelog is missing too.